### PR TITLE
updating URL link that's broken

### PR DESCRIPTION
**Pull request by cmastr**
### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4387.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md [hunk 1]
```diff
@@ -50,6 +50,6 @@ To use the CLI to enable or disable applications, see [Application Deployment](.
 
 ## Upgrade Platform applications from the CLI
 
-Platform Applications within a Project are automatically upgraded when the Workspace that a Project belongs to is upgraded. See [Upgrading Workspace Platform Applications][upgrade_workspaces] for more information on how to upgrade these applications.
+Platform Applications within a Project are automatically upgraded when the Workspace that a Project belongs to is upgraded. See [Upgrade Kommander][upgrade_workspaces] for more information on how to upgrade these applications.
```
Comment (mmcelroy91) [inline pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md:53]: I'm still getting a 404 not found when I click this link on Dev-docs

**Review by mmcelroy91 — CHANGES_REQUESTED** [review]
I'm still getting "sorry not found"

File: pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md (hunk 1 above)
Comment (cmastr) [reply pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md:53]: alright, I suck at this. One more try.

**Review by gflorinskaya — APPROVED** [review]

File: pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md (hunk 1 above)
Comment (cmastr) [reply pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md:53]: Fixed!!

Comment (mmcelroy91) [reply pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md:53]: There it goes!!! Woohoo

**Review by mmcelroy91 — APPROVED** [review]

**Pull request merged**